JobConfig/cosmic/cosmic_s1_dsstops.fcl

```diff
@@ -229,86 +27,28 @@ physics :
          mcTrajectoryInputs : [ "g4run" ]
          vetoDaughters: []
       }
-
-      g4status:
-      {
-         module_type: FilterStatusG4
-         input: "g4run"
-      }
-
-      g4consistent: {
-         module_type: FilterStatusG4
-         input: "g4run"
-	 maxAcceptedStatus: 9  #  status 10 and above means StepPointMCCollection may have non-dereferencable pointers
-      }
+     g4status: @local::Primary.filters.g4status
    }
-
-   trigFilter     : [generate, genCounter, g4run, g4consistent, stepPointMomentumFilter, cosmicFilter, compressPV]
-   g4StatusFilter : [generate, genCounter, g4run, "!g4status", compressPVFull]
-   trigger_paths  : [trigFilter, g4StatusFilter]
-
-   outputs: [truncatedEvtsOutput, filteredOutput]
-   an: [ genCountLogger, collectionSizes, statusG4 ]
-   end_paths: [outputs, an]
+   TriggerPath : [generate, genCounter, g4run, g4status, stepPointMomentumFilter, cosmicFilter]
 }
 
-outputs:
-{
-
-   filteredOutput :
-   {
-      module_type : RootOutput
-      SelectEvents: ["trigFilter"]
-      outputCommands:
-      [
-         "drop *_*_*_*",
-         "keep mu2e::GenParticles_*_*_*",
-         "keep mu2e::GenEventCount_*_*_*",
-         "keep mu2e::StatusG4_*_*_*",
-         "keep mu2e::CosmicLivetime_*_*_*",
-         "keep *_cosmicFilter_*_*",
-         "keep *_compressPV_*_*"
-      ]
-   }
-
-   truncatedEvtsOutput :
-   {
-      module_type : RootOutput
-      SelectEvents: ["g4StatusFilter"]
-      outputCommands:
-      [
-         "drop *_*_*_*",
-         "keep mu2e::GenParticles_*_*_*",
-         "keep mu2e::GenEventCount_*_*_*",
-         "keep mu2e::StatusG4_*_*_*",
-         "keep mu2e::CosmicLivetime_*_*_*",
-         "keep *_g4run_*_*",
-         "drop uintmu2e::PhysicalVolumeInfomvstd::pairs_g4run_*_*",
-         "keep *_compressPVFull_*_*"
-      ]
-   }
+outputs.Output.outputCommands : [@sequence::Primary.KeptProducts, "keep *_cosmicFilter_*_*"]
+services.TFileService.fileName  : "nts.owner.cosmic-s1-dsstops.version.sequencer.root"
```

Comment:
We are eliminating nts output for production files unless they serve a demonstrated purpose that cannot be performed on art output.  to disable nts output, assign the filename to '/dev/null'.

Reply:
I've deleted the the lines 'services.TFileService.fileName'.
Filenames are set to null in primary/prmary.fcl, which I include.